### ISSUE TITLE
Collaborative real-time editor

### ISSUE THREAD

**Issue by thbkrkr** [cityzendata/warp10-quantum#37]
It would be great if the editor is collaborative in real time.

**Comment by StevenLeRoux** [comment]
I agree. It would be great for remote support instead of sharing URLs.

**Comment by hbs** [comment]
Like having an etherpad as the editing area?

**Comment by LostInBrittany** [comment]
I like the idea, for certain use cases, not as default.
I am going to look around for technical solutions that
could fit and do a quick proof of concept, don't hesitate to
make suggestions.
